contract/vm.go

```diff
@@ -52,6 +53,8 @@ const (
 	maxCallDepth         = 64
 	checkFeeDelegationFn = "check_delegation"
 	constructor          = "constructor"
+
+	vmTimeoutErrMsg = "contract timeout of VmTimeoutError"
```

Comment:
"of" or "or"?

it can also be aligned

Reply:
I was not sure how to describe this error effectively. I will change the message soon.

Reply:
if this error message can appear on receipts, then it should not contain "VmTimeoutError"
if used only for logs, it is OK